src/ucp/wireup/select.c

```diff
@@ -1865,26 +1906,34 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
          * Allow selecting additional lanes in case the remote memory will not be
          * registered with this memory domain, i.e with GPU memory.
          */
+        ucp_wireup_dev_usage_count_init(&dev_count[0]);
```

Comment:
use &dev_count[UCS_MEMORY_TYPE_HOST], or define another variable - rkey_ptr_dev_count - specifically for using it in rkey_ptr lane selection

Reply:
Done.